app/global-error.tsx

```diff
@@ -17,4 +22,29 @@ const GlobalErrorPage: FC<{ error: Error }> = ({ error }) => {
   );
 };
 
-export default GlobalErrorPage;
+const GlobalErrorPage: FC<{ error: Error }> = ({ error }) => {
+  useMemo(() => captureException(error), [error]);
+
+  return (
+    <html>
+      <body>
+        <CenteredLayout>
```

Comment:
The BaseLayout should probably be added here before the CenteredLayout, because the BaseLayout gets injected by the `layout.tsx` under `app/[locale]` but the global error page doesn't use our layout.

Hence, the need of BaseLayout surrounding the CenteredLayout

Reply:
I noticed it while checking the error / not-found pages, but I didn't know where it came from. Thanks 👍